torchmetrics/text/eed.py

```diff
@@ -0,0 +1,82 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable, Optional
+
+from torch import tensor
+
+from torchmetrics.functional.text.eed import _eed_compute, _eed_update
+from torchmetrics.metric import Metric
+
+
+class EED(Metric):
+    """Computes extended edit distance score (`EED`_) [1] for strings or list of strings The metric utilises the
+    Levenshtein distance and extends it by adding an additional jump operation.
+
+    Args:
+        language: Language used in sentences. Only supports English (en) and Japanese (ja) for now. Defaults to en
+
+    Returns:
+        Extended edit distance score as a tensor
+
+    Example:
+        >>> hypotheses = ["this is the prediction", "here is an other sample"]
+        >>> references = ["this is the reference", "here is another one"]
+        >>> metric = EED()
+        >>> metric(hypotheses=hypotheses, references=references)
+        tensor(0.3078)
+
+    References:
+        [1] P. Stanchev, W. Wang, and H. Ney, “EED: Extended Edit Distance Measure for Machine Translation”, submitted
+        to WMT 2019. `EED`_
```

Comment:
```suggestion
        [1] EED: Extended Edit Distance Measure for Machine Translation by P. Stanchev, W. Wang, and H. Ney `EED`_
```

Reply:
My previous citation doesn't follow the normal way of citing papers like other citations in this library, but I wrote it that way because the license asked me to. Ignoring that part of the license should be ok though, right?

Reply:
@Borda ^^

Reply:
@mathemusician could you please refer to where the license specifies the reference formatting?
if it is really needed we can keep the requested format :rabbit:

Reply:
@Borda [https://github.com/rwth-i6/ExtendedEditDistance/blob/master/LICENSE#L28-L29](https://github.com/rwth-i6/ExtendedEditDistance/blob/master/LICENSE#L28-L29)

Reply:
yeah, that is completely correct to cite the publication
cc: @justusschock